pages/settings/wallets/index.js

```diff
@@ -1,29 +1,54 @@
 import { getGetServerSideProps } from '@/api/ssrApollo'
 import Layout from '@/components/layout'
 import styles from '@/styles/wallet.module.css'
-import { WalletCard } from '@/components/wallet-card'
-import { LightningAddressWalletCard } from './lightning-address'
-import { LNbitsCard } from './lnbits'
-import { NWCCard } from './nwc'
-import { LNDCard } from './lnd'
-import { CLNCard } from './cln'
-import { WALLETS } from '@/fragments/wallet'
-import { useQuery } from '@apollo/client'
-import PageLoading from '@/components/page-loading'
-import { LNCCard } from './lnc'
 import Link from 'next/link'
-import { Wallet as W } from '@/lib/constants'
+import { useWallets } from '@/components/wallet'
+import { useState } from 'react'
+import dynamic from 'next/dynamic'
 
-export const getServerSideProps = getGetServerSideProps({ query: WALLETS, authRequired: true })
+const WalletCard = dynamic(() => import('@/components/wallet-card'), { ssr: false })
+
+export const getServerSideProps = getGetServerSideProps({ authRequired: true })
 
 export default function Wallet ({ ssrData }) {
-  const { data } = useQuery(WALLETS)
+  const { wallets } = useWallets()
+
+  const [sourceIndex, setSourceIndex] = useState()
+  const [targetIndex, setTargetIndex] = useState()
```

Comment:
I hope this drag and drop feature is still in line with our wallets plans but I think it's cool.

Might need to add some explanation what drag & drop does. It configures the priority per wallet so if a wallet fails, the next wallet will be tried.

It's only used during autowithdrawals yet though. The `useWallet` hook on the client does not support multiple enabled wallets yet. It will only try the first enabled.